data/genesis/decaf.toml

```diff
@@ -1,5 +1,7 @@
 base_version = "0.2"
 upgrade_version = "0.3"
+epoch_height = 3000
+epoch_start_block = 3102001
```

Comment:
Can someone double check the math on this start block too? More eyes is better I think